annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -91,7 +91,7 @@
         {"Thr", "T"}, {"Trp", "W"}, {"Tyr", "Y"}, {"Val", "V"}, {"Xxx", "X"}, {"Ter", "*"}
     };
     private Set<String> spliceSiteVariants = new HashSet<String>(Arrays.asList(
-            "splice_acceptor_variant", "splice_donor_variant", "splice_region_variant"));
+            "splice_acceptor_variant", "splice_donor_variant", "splice_region_variant", "stop_lost"));
```

Comment:
`stop_lost` is not a type of spliceSiteVariant. Better to handle it separately or not at all, as described below.